vars/promoteArtifacts.groovy

```diff
@@ -29,55 +28,57 @@ void call(Map args = [:]) {
 
     print("Actions ${fileActions}")
 
+    argsMap = [:]
+    argsMap['signatureType'] = '.sig'
+
+    //////////// Signing Artifacts
+    println("Signing Core Pluings")
+    String corePluginDir = "$WORKSPACE/artifacts/$artifactPath/builds/$filename/core-plugins"
+    argsMap['artifactPath'] = corePluginDir
+    for (Closure action : fileActions) {
+        action(argsMap)
+    }
+
+    println("Signing TAR Artifacts")
+    String coreFullPath = ['core', filename, version].join('/')
+    String bundleFullPath = ['bundle', filename, version].join('/')
+    for (Closure action : fileActions) {
+        for (file in findFiles(glob: "**/${filename}-min-${version}*.tar.gz,**/${filename}-${version}*.tar.gz")) {
+            argsMap['artifactPath'] = "$WORKSPACE" + "/" + file.getPath()
+            action(argsMap)
+        }
+    }
+
+
```

Comment:
A lot of empty lines, remove, and maybe just do a `println` instead of `Uploading Artifacts`

On that note, elsewhere we use `echo` and not println, consolidate.

Reply:
Maybe we should start using println instead of echo.... using more groovy?